.travis.yml

```diff
@@ -84,3 +74,26 @@ script:
 after_script:
     - echo $TRAVIS_COMMIT_RANGE
     - echo $TRAVIS_COMMIT_LOG
+
+jobs:
+  include:
+    - stage: check_doc
```

Comment:
How about add `sudo: false` here, travis can boot faster

Reply:
Please see my previous diff to get it to work

Reply:
https://github.com/bitcoin/bitcoin/pull/13183#discussion_r186475302

Reply:
Done